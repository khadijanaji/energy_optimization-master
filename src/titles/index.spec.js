/* globals describe it expect */

import {getCategoryObject} from "./";

describe("Helper", () => {
  describe("Titles resolver", () => {
    it("Get pap.production category object", () => {
      expect(getCategoryObject("pap", "production", "")).toMatchSnapshot();
    });
  });
});
