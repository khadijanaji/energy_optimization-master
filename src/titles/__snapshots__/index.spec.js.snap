// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Helper Titles resolver Get pap.production category object 1`] = `
Object {
  "indicators": undefined,
}
`;
